### DISCUSSION TITLE
OpenChain ISO/IEC 5230 Open Source Policy Template Now In Simplified Chinese

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#63] [TODO & OSPO Alliance chapter]
From TODO Group on Slack: https://thetodogroup.slack.com/archives/CLXQQNVB6/p1647995067401519

OpenChain ISO/IEC 5230 Open Source Policy Template Now In Simplified Chinese
One of the key documents in the OpenChain ecosystem – and the broader world of open source license compliance – is our policy template. This is designed to help organizations quickly, effectively create open source policies for their teams.
The policy template was originally created by Andrew Katz at Moorcrofts in the United Kingdom. It has since been explored and used by open source stakeholders around the world. Recently, the OpenChain China Work Group began to explore localization, and thanks to the hard work of Chloe Zhong from Honor we now have a complete document in Simplified Chinese to share with the world.

Learn more:
https://www.openchainproject.org/featured/2022/03/22/policy-template-zh-hans

**Comment by wangleo61** [comment]
给力
